pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -236,6 +236,9 @@ let
     # arbitrarily set to the current latest bazel version, overly careful
     TF_IGNORE_MAX_BAZEL_VERSION = true;
 
+    # aarch64-darwin needs this env set
```

Comment:
```suggestion
```
This is needed on x86_64-darwin as well, see #192123

Reply:
Thanks